packages/salesforcedx-vscode-lwc/test/vscode-integration/jsconfig.test.ts

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as fs from 'fs';
+import * as path from 'path';
+import { Position, window, workspace, WorkspaceEdit } from 'vscode';
+import URI from 'vscode-uri';
+
+const CONFIG_FILENAME = 'jsconfig.json';
+const TEST_COMPONENT_NAME = 'testComponent';
+const CREATE_COMPONENT_NAME = 'createComponent';
+
+describe('jsconfig Test Suite', () => {
+  let modulesDir: string;
+  let configPath: string;
+  let config: object;
+
+  before(async () => {
+    modulesDir = path.join(
+      workspace.workspaceFolders![0].uri.fsPath,
+      'force-app',
+      'main',
+      'default',
+      'lwc'
+    );
+    configPath = path.join(modulesDir, CONFIG_FILENAME);
+  });
+
+  beforeEach(async function() {
```

Comment:
async function() ?
You did async () => {} above

Reply:
Nice catch. I had to move away from the arrow functions to set the test case timeout.

Reply:
ahh right.